pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.black]
+line-length = 119
+target-version = ['py38']
+
+[tool.pytest.ini_options]
+filterwarnings = ["ignore:the imp module is deprecated in favour of importlib:DeprecationWarning:tensorflow.*"]
+
+[tool.poetry]
+name = "evaluation"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.8.11"
```

Comment:
`python>=3.8.10` supports Apple Silicon and macOS BigSur.
Personally I'm not using them. Just to ensure no one will got surprised by an incompatible version.